### ISSUE TITLE
Some plugins become undefined due to ordering of cordova plugin_list

### ISSUE THREAD

**Issue by CFT-Chris** [ionic-team/capacitor#1615]
**Description of the problem:** Like the title states, the new ordering of plugin_list causes some plugins to become undefined. If there is a merge sharing the same Cordova namespace as a clobber, then the merge gets clobbered.  launchnavigator is one such plugin that has a merge (for platform specific JS code) with a clobber (its `common.js` code).

Affected platform
- [x] Android
- [x] iOS
- [ ] electron
- [ ] web

OS of the development machine
- [x] Windows
- [ ] macOS
- [ ] linux

**Capacitor version:** 1.0.0, but bug exists since beta-25

**Steps to reproduce:**

ionic start foobar blank --capacitor
ionic build
npm i uk.co.workingedge.phonegap.plugin.launchnavigator
npx cap add android
npx cap open android

Project sync, build and run.

**Issue:** 
launchnavigator (or whatever plugin that relies on merges and clobbers) is undefined

I have a PR to fix this. Prior to #1524, the merge into the launchnavigator namespace happens after the clobber. After #1524, the merge becomes clobbered instead because `clobberKey = ""` comes alphabetically before `clobberKey = "launchnavigator"`, and launchnavigator is thus missing everything in `plugins/uk.co.workingedge.phonegap.plugin.launchnavigator/www/android/launchnavigator.js`.

**Comment by jcesarmobile** [comment]
Looks like the Fixes: didn't work. Closing as #1616 is merged.
